rero_ils/modules/documents/serializers.py

```diff
@@ -30,6 +30,92 @@
 from ..organisations.api import Organisation
 from ..serializers import JSONSerializer, RecordSchemaJSONV1
 
+DOCUMENT_TYPES = {
+    'docmaintype_archives': {},
+    'docmaintype_article': {},
+    'docmaintype_audio': {
+        'docsubtype_music': True,
+        'docsubtype_sound': True,
+        'docsubtype_audio_book': True,
+        'docsubtype_recorded_words': True
+    },
+    'docmaintype_book': {
+        'docsubtype_other_book': True,
+        'docsubtype_large_print_book': True,
+        'docsubtype_braille_book': True,
+        'docsubtype_manuscript': True,
+        'docsubtype_thesis': True,
+        'docsubtype_e-book': True,
+        'docsubtype_microform': True
+    },
+    'docmaintype_children': {},
+    'docmaintype_comic': {
+        'docsubtype_comic': True,
+        'docsubtype_manga': True
+    },
+    'docmaintype_electronic': {
+        'docsubtype_cdrom_dvdrom': True,
+        'docsubtype_other_electronic_carrier': True
+    },
+    'docmaintype_game': {
+        'docsubtype_other_game': True,
+        'docsubtype_video_game': True,
+        'docsubtype_role-playing_game': True,
+        'docsubtype_board_game': True
+    },
+    'docmaintype_image': {
+        'docsubtype_poster': True,
+        'docsubtype_postcard': True,
+        'docsubtype_photography': True,
+        'docsubtype_print_engraving': True,
+        'docsubtype_calendar': True,
+        'docsubtype_other_image': True
+    },
+    'docmaintype_language_method': {},
+    'docmaintype_leaf': {},
+    'docmaintype_map': {
+        'docsubtype_map': True,
+        'docsubtype_atlas': True
+    },
+    'docmaintype_movie_series': {
+        'docsubtype_movie': True,
+        'docsubtype_series': True,
+        'docsubtype_documentary': True
+    },
+    'docmaintype_object': {
+        'docsubtype_kamishibai': True,
+        'docsubtype_tablet': True,
+        'docsubtype_other_object': True
+    },
+    'docmaintype_other': {},
+    'docmaintype_score': {
+        'docsubtype_printed_score': True,
+        'docsubtype_music_method': True,
+        'docsubtype_manuscript_score': True
+    },
+    'docmaintype_serial': {},
+    'docmaintype_series': {}
+}
+
+
+def filter_document_type_buckets(buckets):
```

Comment:
It seems a kind of private function. Python best practice are to prepend function name with "_" in this case

Reply:
It also used in item serializer and maybe in import serializer. I have put everythin into utils.py.